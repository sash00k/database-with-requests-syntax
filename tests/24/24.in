get records where (student_id is set and department is "ММФ") as students_mmf
get records from students_mmf where (group in (get records where name is "Кузнецова Л. И.").groups and group is 8126)