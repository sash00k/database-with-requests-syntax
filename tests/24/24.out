Student Зуброва Д. А. from group 8126, ММФ